examples/producer/src/main/java/io/strimzi/examples/producer/ExampleProducer.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2017-2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.examples.producer;
+
+import io.strimzi.kafka.oauth.client.ClientConfig;
+import io.strimzi.kafka.oauth.common.Config;
+import io.strimzi.kafka.oauth.common.ConfigProperties;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+public class ExampleProducer {
+
+    public static void main(String[] args) {
+
+        String topic = "Topic1";
+
+        Properties defaults = new Properties();
+        Config external = new Config();
+
+        // TODO: Set KEYCLOAK_IP to be able to connect to Keycloak
+        //  Use 'keycloak.ip' system property or KEYCLOAK_IP env variable
```

Comment:
as for the consumer part I guess you can remove this TODO. The code does it now.